migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc

```diff
@@ -20,30 +20,44 @@ include::modules/ossm-cluster-wide-migration-methods.adoc[leveloffset=+1]
 // The following tasks are for migration using the Istio revision label
 include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-revision-label.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../install/ossm-sidecar-injection-assembly.adoc#ossm-identifying-revision-name_ossm-sidecar-injection-assembly[Identifying the revision name]
-
 include::modules/ossm-migrating-workloads-using-the-istio-revision-label.adoc[leveloffset=+2]
 
-include::modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc[leveloffset=+1]
 
 .Next steps
 
-You can proceed to complete the migration and remove {SMProduct} 2 resources.
+If you are using gateways, you must migrate them before you complete the migration process.
+
+* xref:../../migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc[Migrating gateways from Service Mesh 2 to Service Mesh 3]
+
+If you are not using gateways, and have verified your cluster-wide migration, you can proceed to complete the migration and remove OpenShift Service Mesh 2 resources.
 
 * xref:../../migrating/done/ossm-migrating-complete-assembly.adoc[Completing the Migration]
 
 // The following tasks are for migration using the Istio injection label
 include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
+include::modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc[leveloffset=+2]
 
-* xref:../../install/ossm-sidecar-injection-assembly.adoc#ossm-identifying-revision-name_ossm-sidecar-injection-assembly[Identifying the revision name]
+.Next steps
 
-include::modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc[leveloffset=+2]
+If you are using gateways, you must migrate them before you complete the migration process.
+
+* xref:../../migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc[Migrating gateways from Service Mesh 2 to Service Mesh 3]
+
+If you are not using gateways, and have verified your cluster-wide migration, create a default revision tag and re-label namespaces.
```

Comment:
This is one word, without a hyphen in the [next section](https://github.com/openshift/openshift-docs/pull/89488/files#diff-39de4acdcc4dafaa0c62b6f1214e6202d4ec52566b0dd6fca541342ff63a0f5eR7).

```suggestion
If you are not using gateways, and have verified your cluster-wide migration, create a default revision tag and relabel namespaces.
```

Reply:
thank you for catching this!

Reply:
done